prody/proteins/compare.py

```diff
@@ -17,8 +17,7 @@
     range = xrange
 
 __all__ = ['matchChains', 'matchAlign', 'mapOntoChain',
-           'mapChainByChain', 'mapOntoChainByAlignment', 
-           'getMatchScore', 'setMatchScore',
+           'mapChainByChain', 'getMatchScore', 'setMatchScore',
```

Comment:
mapOntoChainByAlignment was added in #537

Reply:
This got deleted by accident. I'm restoring it back